### PR TITLE
Add Unit tests for Util Classes

### PR DESCRIPTION
**Pull request by siddharth2010**

### CODE REVIEW

File: app/src/test/java/org/kiwix/kiwixmobile/utils/NetworkUtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+  public void testNetworkAvailability() {
+    NetworkInfo[] info = {networkInfo1,networkInfo2};
+    when(context.getSystemService(Context.CONNECTIVITY_SERVICE)).thenReturn(connectivity);
+    when(connectivity.getAllNetworkInfo()).thenReturn(info);
+
+    //one network is connected
+    when(networkInfo1.getState()).thenReturn(NetworkInfo.State.CONNECTED);
+    when(networkInfo2.getState()).thenReturn(NetworkInfo.State.DISCONNECTING);
+    assertEquals("",true,NetworkUtils.isNetworkAvailable(context));
```
Comment (mhutti1) [inline app/src/test/java/org/kiwix/kiwixmobile/utils/NetworkUtilsTest.java]: There are a couple places where there are no spaces after commas.

Comment (siddharth2010) [reply app/src/test/java/org/kiwix/kiwixmobile/utils/NetworkUtilsTest.java]: @mhutti1 , I have fixed this and updated the branch.

Comment (siddharth2010) [reply app/src/test/java/org/kiwix/kiwixmobile/utils/NetworkUtilsTest.java]: @mhutti1 , the Travis build is failing and I have no idea why. It's working perfectly on my local machine.

**Comment by siddharth2010** [comment]
@mhutti1 , I have fixed this and updated the branch, but the Travis build is failing and I have no idea why. It's working perfectly on my local machine.

File: app/src/test/java/org/kiwix/kiwixmobile/utils/NetworkUtilsTest.java (hunk 1 above)
Comment (siddharth2010) [reply app/src/test/java/org/kiwix/kiwixmobile/utils/NetworkUtilsTest.java]: @mhutti1 , The build passed on its own, after restarting it right now. So I think its all good.

**Pull request merged**